modules/db2/src/test/java/org/testcontainers/junit/db2/SimpleDb2ContainerTest.java

```diff
@@ -10,7 +10,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class SimpleDb2Test extends AbstractContainerDatabaseTest {
+public class SimpleDb2ContainerTest extends AbstractContainerDatabaseTest {
```

Comment:
Renamed this to keep it consistent with the rest - I can move this out of this current PR if you wish.

Reply:
I think we can live with or without it :D For example `SimpleMySQLTest` or `SimpleMSSQLServerTest` haven't changed :eyes:

Reply:
- [ ] undo rename